addons/web_editor/static/src/js/editor/odoo-editor/test/spec/color.test.js

```diff
@@ -1,9 +1,9 @@
 import { BasicEditor, testEditor, unformat } from '../utils.js';
 import { rgbToHex } from '../../src/utils/utils.js';
 
-const setColor = (color, mode) => {
+const setColor = (color, mode, opacity) => {
     return async editor => {
-        await editor.execCommand('applyColor', color, mode);
+        await editor.execCommand('applyColor', color, mode, undefined, opacity);
```

Comment:
@adsa-odoo sending undefined what purpose does it serve?

Reply:
@shsa-odoo in `applyColor` method, we do have 4th argument `element`. But we want to skip that argument here in color.test.js file as we don't need that but we need 5th argument. That's why I put 4th argument as undefined.